streams/src/main/java/org/apache/kafka/streams/kstream/KStream.java

```diff
@@ -721,13 +725,33 @@ void writeAsText(final String filePath,
      *                    be used
      * @param topic       the topic name
      * @return a {@code KStream} that contains the exact same (and potentially repartitioned) records as this {@code KStream}
+     * @deprecated use {@code through(String, Produced)}
      */
+    @Deprecated
     KStream<K, V> through(final Serde<K> keySerde,
                           final Serde<V> valSerde,
                           final StreamPartitioner<? super K, ? super V> partitioner,
                           final String topic);
 
     /**
+     * Materialize this stream to a topic and creates a new {@code KStream} from the topic using the
+     * {@link Produced} instance for configuration of the key {@link Serde key serde}, value {@link Serde value serde},
```

Comment:
This would print as `key key serde` where `key serde` part has the link. Maybe just write as

```
of the {@link Serde key serde}, {@link Serde value serde}, ...
```

ditto else where.